src/renderer/components/+namespaces/namespace.store.ts

```diff
@@ -34,7 +34,7 @@ export function getDummyNamespace(name: string) {
 export class NamespaceStore extends KubeObjectStore<Namespace> {
   api = namespacesApi;
 
-  @observable contextNs = observable.array<string>();
+  @observable private contextNs = observable.set<string>();
```

Comment:
Making it as `private` and updating to `observable.set` is a breaking change.
Public version for getting context namespaces is `NamespaceStore.getContextNamespaces(): string[]`